Whiskey/Tasks/Invoke-WhiskeyPowerShell.ps1

```diff
@@ -58,7 +58,22 @@ function Invoke-WhiskeyPowerShell
 
         if( -not (Test-Path -Path $WorkingDirectory -PathType Container) )
         {
-            Stop-WhiskeyTask -TaskContext $TaskContext -Message ('Can''t run PowerShell script ''{0}'': working directory ''{1}'' doesn''t exist.' -f $ScriptPath,$WorkingDirectory)
+            Stop-WhiskeyTask -TaskContext $TaskContext -Message ('Can''t run PowerShell script ''{0}'': working directory ''{1}'' doesn''t exist.' -f $scriptPath,$WorkingDirectory)
+        }
+
+        $scriptCommand = Get-Command -Name $scriptPath -ErrorAction Ignore
+        if( -not (Test-Path -Path $WorkingDirectory -PathType Container) )

```

Comment:
This `if` statement is not correct based on the `Stop-WhiskeyTask` message it throws.